BIDebugEngine/BIDebugEngine/RVClasses.cpp

```diff
@@ -319,22 +319,28 @@ sourcedocpos tryGetFilenameAndCode(const intercept::types::vm_context::callstack
 
     auto& type = typeid(*&it);
     //#TODO X64 fails on this
-    __debugbreak(); //Some place already has these
+    // __debugbreak(); //Some place already has these
     const auto typeName = ((&type) && ((__std_type_info_data*) &type)->_UndecoratedName) ? type.name() : "TypeFAIL";
     auto hash = type.hash_code();
 
     switch (hash) {
         case CallStackItemSimple::type_hash: {
             auto stackItem = static_cast<const CallStackItemSimple*>(&it);
             //#TODO test if this is the correct instruction or if i should -1 this
-            const auto& lastInstruction = (stackItem->_instructions.get(stackItem->_currentInstruction))->sdp;
-            return lastInstruction;
+            auto instructionIdx = stackItem->_currentInstruction - 1;
+            if(instructionIdx >= 0 && instructionIdx < stackItem->_instructions.size())
+            {
```

Comment:
wrong brace layout

Reply:
Fixed in 137ea6ee36d15d5daf674182028ddd7d52361a64.